From fbdf8536abd134de206cbae097784fe1a78df02a Mon Sep 17 00:00:00 2001
From: Paul Menzel <pmenzel@molgen.mpg.de>
Date: Tue, 23 Apr 2019 17:11:51 +0200
Subject: [PATCH 13/29] NEWS: List getty@tty1.service as not being enabled
 anymore

Fixes: https://github.com/systemd/systemd/issues/12345
(cherry picked from commit 1fa3ba909605c13761af69ae0ba1653292b6e730)
---
 NEWS | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/NEWS b/NEWS
index f7b145f20e..2cf96c2a82 100644
--- a/NEWS
+++ b/NEWS
@@ -221,8 +221,9 @@ CHANGES WITH 242:
           a different layout of the bootloader partitions (for example grub2).
 
         * During package installation (with `ninja install`), we would create
-          symlinks for systemd-networkd.service, systemd-networkd.socket,
-          systemd-resolved.service, remote-cryptsetup.target, remote-fs.target,
+          symlinks for getty@tty1.service, systemd-networkd.service,
+          systemd-networkd.socket, systemd-resolved.service,
+          remote-cryptsetup.target, remote-fs.target,
           systemd-networkd-wait-online.service, and systemd-timesyncd.service
           in /etc, as if `systemctl enable` was called for those units, to make
           the system usable immediately after installation. Now this is not
-- 
2.21.0

